<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PomodoroApp</class>
 <widget class="QMainWindow" name="PomodoroApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PomodoroApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>381</width>
      <height>230</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="font">
          <font>
           <family>FiraCode Nerd Font</family>
          </font>
         </property>
         <property name="text">
          <string>Start pomodoro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="made_by">
         <property name="font">
          <font>
           <family>FiraCode Nerd Font</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Made with ❤️ blitty </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
       <item>
        <widget class="QLCDNumber" name="lcdHour"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>AR PL UKai CN</family>
           <pointsize>42</pointsize>
          </font>
         </property>
         <property name="text">
          <string>:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdMin"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="format">
         <property name="font">
          <font>
           <family>FiraCode Nerd Font</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>FiraCode Nerd Font</family>
            </font>
           </property>
           <property name="text">
            <string>Number of pomodoros:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNPomodoros"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_Pomodoro"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionSet_Pomodoro">
   <property name="text">
    <string>Set Pomodoro</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
